app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] [REQUIRED] Good job so far! Please the `:confirmable` should be enabled in the `user` model (`user.rb` file), uncomment that🙏🏼      
 You can see something similar to what is below:

```ruby
	devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```